services/121-service/src/payments/dto/reconciliation-feedback.dto.ts

```diff
@@ -4,7 +4,7 @@ import { TransactionStatusEnum } from '@121-service/src/payments/transactions/en
 import { ImportStatus } from '@121-service/src/registration/dto/bulk-import.dto';
 
 export class ReconciliationFeedbackDto {
-  // ##TODO align wigh guidelines: use the dto only for the endpoint response, and don't pass it along multiple methods > create new interfaces for that + split in files where appropriate
```

Comment:
I think we can leave this as a normal TODO, we can do later. As this was never alligned before

Reply:
I think Diderik will look at this in the his guidelines review task.